sdk/apimanagement/api-management-custom-widgets-tools/review/api-management-custom-widgets-tools.api.md

```diff
@@ -0,0 +1,68 @@
+## API Report File for "@azure/api-management-custom-widgets-tools"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+// @public
+export const ASK_FOR_SECRETS_MESSAGE_KEY = "askForSecretsMSAPIM";
+
+// @public
+export const askForSecrets: (targetOrigin: string, instanceId: string, targetModule: TTargetModule, environment: TEnvironment) => Promise<TSecrets>;
+
+// @public
+export function buildOnChange<TValues extends TValuesBase>(valuesDefault: TValues): TOnChange<TValues>;
+
+// @public
+export function deployNodeJS(serviceInformation: TServiceInformation, name: string, fallbackConfigPath?: string): Promise<void>;
+
+// @public
+export const EDITOR_DATA_KEY = "editorData";
+
+// @public
+export function getEditorData<TValues extends TValuesBase>(valuesDefault: TValues): TEditorData<TValues>;
+
+// @public
+export function getEditorValues<TValues extends TValuesBase>(valuesDefault: TValues): TValues;
+
+// @public
+export const ON_CHANGE_MESSAGE_KEY = "customInputValueChangedMSAPIM";
+
+// @public
+export function onChangeWithOrigin<TValues extends TValuesBase>(origin: TEditorData<TValues>["origin"], instanceId: TEditorData<TValues>["instanceId"], values: TValues): void;
+
+// @public
+export interface TEditorData<TValues extends TValuesBase> {
+    environment: TEnvironment;
+    instanceId: string;
+    origin: string;
+    values: TValues;
+}
+
+// @public
+export type TEnvironment = "development" | "publishing" | "runtime" | "error";
+
+// @public
+export type TOnChange<TValues extends TValuesBase> = (values: Partial<TValues>) => void;
+
+// @public
+export type TSecrets = {
+    token: string;
+    userId: string;
+};
+
+// @public
+export type TServiceInformation = {
+    resourceId: string;
+    managementApiEndpoint: string;
+    apiVersion?: string;
+    tokenOverride?: string;
+};
+
+// @public
+export type TTargetModule = "app" | "editor";
+
+// @public
+export type TValuesBase = Record<string, unknown>;
```

Comment:
Let's drop `Base`, which is a common suffix for abstract classes in C# but not idiomatic for Typescript, also this is a type not a class. `Values` or `ValuesCommon`